README.md

```diff
@@ -1,143 +1,431 @@
+│ 🟫🟫🟫🟨🟨 │     41 │   0.0177106 │
+│ 🟫🟫🟫🟨🟩 │     17 │  0.00734341 │
+│ 🟫🟫🟫🟩🟫 │     17 │  0.00734341 │
+│ 🟫🟫🟫🟩🟨 │      9 │  0.00388769 │
+│ 🟫🟫🟫🟩🟩 │     20 │  0.00863931 │
+│ 🟫🟫🟨🟫🟫 │    107 │   0.0462203 │
+│ 🟫🟫🟨🟫🟨 │     35 │   0.0151188 │
+│ 🟫🟫🟨🟫🟩 │     25 │   0.0107991 │
+│ 🟫🟫🟨🟨🟫 │     21 │  0.00907127 │
+│ 🟫🟫🟨🟨🟨 │      4 │  0.00172786 │
+│ 🟫🟫🟨🟨🟩 │      5 │  0.00215983 │
+│ 🟫🟫🟨🟩🟫 │      6 │  0.00259179 │
+│ 🟫🟫🟨🟩🟨 │      0 │         0.0 │
+│ 🟫🟫🟨🟩🟩 │      0 │         0.0 │
+│ 🟫🟫🟩🟫🟫 │     51 │   0.0220302 │
+│ 🟫🟫🟩🟫🟨 │     15 │  0.00647948 │
+│ 🟫🟫🟩🟫🟩 │     23 │   0.0099352 │
+│ 🟫🟫🟩🟨🟫 │     29 │    0.012527 │
+│ 🟫🟫🟩🟨🟨 │      3 │   0.0012959 │
+│     ⋮      │   ⋮    │      ⋮      │
+└────────────┴────────┴─────────────┘
+                     220 rows omitted
 ```
 
-benchmarking play of a random Wordle game produces
+Assuming the targets are equally likely, which apparently is the case in the online games, the probability of each score is the count for that score divided by size of the active target pool.
+The expected pool size is the sum of the `counts` multiplied by the `probs` or, equivalently, the sum of the squared counts divided by the sum of the counts.
 
 ```jl
-julia> @benchmark playgame($wordle)
-BenchmarkTools.Trial: 10000 samples with 1 evaluation.
- Range (min … max):   3.410 μs …  7.382 ms  ┊ GC (min … max): 0.00% … 98.92%
- Time  (median):     70.059 μs              ┊ GC (median):    0.00%
- Time  (mean ± σ):   85.787 μs ± 89.935 μs  ┊ GC (mean ± σ):  0.85% ±  0.99%
-
-      ▂▁▂▃▄▇▇▇▃█▂▆ ▁        ▁      ▂▂▂▁                        
-  ▁▅▆▅████████████▇█▆▄▄▄▄▆▅▇█▆▇▇▇▆▇████▇▇▇▆▆▅▄▂▂▁▁▁▂▁▃▃▃▄▄▄▃▃ ▄
-  3.41 μs         Histogram: frequency by time         211 μs <
-
- Memory estimate: 3.07 KiB, allocs estimate: 50.
- ```
+julia> sum(abs2, wordle.counts) / sum(wordle.counts)  # abs2(x) returns x * x
+61.00086393088553
+```
+
+The next guess is chosen to minimize the expected pool size.
+
+### MAximizing entropy
+
+An alternative criterion is to maximize the [entropy](https://en.wikipedia.org/wiki/Entropy_(information_theory)) of the probabilities associated with the possible scores, i.e. the `probs` field.
+
+Measured in bits, the entropy of the `n` probabilities is $-\sum_{i=1}^n p_i\,log_2(p_i)$
+(The negative sign is why the entropy of the 1-choice probability vector `[1.0]` is `-0.0`, as shown above.)
+
+It measures how the probability is dispersed among the possible scores.
+The best case is for each of the `n` possible scores to have probability `1/n` of occurring.In that case, whichever score is returned, there will only be a small number of targets with that score.
+It is not possible to get equal from a starting guess but, sometimes when the target pool is small, a particular guess may be able to split the remaining `k` targets into `k` distinct scores.
+
+In particular, this always occurs when there are only two targets left.
+
+Chosing a guess so as to maximize the entropy provides slightly better performance, on average.
```

Comment:
This is interesting and it took me a moment to think about why entropy (i.e. expected information) and expected pool size would give different answers. I'm trying to think of something more interesting/expository to say for other readers, but I don't have anything profound other than this is a subtle but cool difference.